onnxoptimizer/passes/set_unique_name_for_node.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// ATTENTION: The code in this file is highly EXPERIMENTAL.
+// Adventurous users should note that the APIs will probably change.
+
+#pragma once
+
+#include "onnxoptimizer/pass.h"
+
+namespace ONNX_NAMESPACE {
+namespace optimization {
+
+struct SetUniqueNameForNode final : public PredicateBasedPass {
+  explicit SetUniqueNameForNode()
+      : PredicateBasedPass(PassType::Other, PassEfficiency::Complete,
+                           PassOptimizationType::None) {}
+
+  std::string getPassName() const override {
+    return "set_unique_name_for_node";
```

Comment:
```suggestion
    return "set_unique_name_for_nodes";
```

Also for the class name

Reply:
done